jvm-runtime/jvm_integration_test.go

```diff
@@ -14,6 +14,22 @@ import (
 	"github.com/alecthomas/repr"
 )
 
+func TestLifecycle(t *testing.T) {
+	in.Run(t,
+		in.WithLanguages("java", "kotlin"),
+		in.GitInit(),
+		in.Exec("rm", "ftl-project.toml"),
+		in.IfLanguage("kotlin", in.Exec("rm", "-r", "echo")), //horrible, but we need to do cleanup, I wonder if we should be running each test in a separate directory
```

Comment:
Oh is the state from the "java" test bleeding over? If so that's a bug, it should be wiping the test directory in between language runs definitely.

Reply:
I have filed https://github.com/TBD54566975/ftl/issues/2412